src/pkg/packaging/dir.props

```diff
@@ -9,26 +9,27 @@
     <SharedHostPublishRoot>$(IntermediateOutputRootPath)sharedHost/</SharedHostPublishRoot>
     <HostFxrPublishRoot>$(IntermediateOutputRootPath)hostFxr/</HostFxrPublishRoot>
     <SharedFrameworkPublishRoot>$(IntermediateOutputRootPath)sharedFx/</SharedFrameworkPublishRoot>
-    <CombinedPublishRoot>$(IntermediateOutputRootPath)combined-framework-host/</CombinedPublishRoot>   
+    <CombinedPublishRoot>$(IntermediateOutputRootPath)combined-framework-host/</CombinedPublishRoot>
   </PropertyGroup>
 
   <PropertyGroup>
     <ProductMoniker>$(PackageTargetRid)-$(HostFullVersion)</ProductMoniker>
+    <PortableMoniker Condition="'$(PortableBuild)' == 'true'">-portable</PortableMoniker>
```

Comment:
Today, we're still building Ubuntu and Debian as non portable. I presume just as validation.